package.json

```diff
@@ -11,7 +11,7 @@
     "@testing-library/user-event": "^7.2.1",
     "bulma": "^0.8.1",
     "eslint-config-google": "^0.14.0",
-    "node-sass": "^4.13.1",
+    "node-sass": "^5.0.0",
```

Comment:
node-sass is deprecated. we should move to dart sass https://sass-lang.com/install